### PR TITLE
Unregister service worker based on config

### PR DESCRIPTION
**Pull request by petrican**
Screenshot 2021-11-01 at 10 46 15

### CODE REVIEW

File: misc/services/mobile-webui/mobile-webui-frontend/src/serviceWorkerRegistration.js [hunk 1]
```diff
@@ -40,7 +42,9 @@ export function register(config) {
     } else {
       // Is not localhost. Just register service worker
       console.log('[ServiceWorker] - Registering on Prod server');
```
Comment (teosarca) [inline misc/services/mobile-webui/mobile-webui-frontend/src/serviceWorkerRegistration.js:44]: i thikn that console.log shall go inside the `if `

File: misc/services/mobile-webui/mobile-webui-frontend/src/serviceWorkerRegistration.js [hunk 2]
```diff
@@ -40,7 +42,9 @@ export function register(config) {
...
-      registerValidSW(swUrl, config);
+      if (REGISTER_SERVICE_WORKER) {
```
Comment (teosarca) [inline misc/services/mobile-webui/mobile-webui-frontend/src/serviceWorkerRegistration.js:45]: why we need this here when we already have an `if (REGISTER_SERVICE_WORKER)` then call this `register` function.

**Comment by teosarca** [comment]
@petrican  another question: how the code will behave in the following scenario:

* we already rolled out with REGISTER_SERVICE_WORKER=true
* now, we decide to disable service workers, wo we set REGISTER_SERVICE_WORKER=false and we do another rollout


Will it unregister currently registered service workers?

**Comment by petrican** [comment]
> @petrican another question: how the code will behave in the following scenario:
> 
> * we already rolled out with REGISTER_SERVICE_WORKER=true
> * now, we decide to disable service workers, wo we set REGISTER_SERVICE_WORKER=false and we do another rollout
> 
> Will it unregister currently registered service workers?

Yes as there is a routine that unregisters those
Screenshot 2021-11-01 at 11 32 29

**Comment by petrican** [comment]
You were right @teosarca I should do one single check and that within the register routine within the serviceWorkerRegistration. Fixed

**Comment by teosarca** [comment]
@petrican  lgtm

**Pull request merged**